<!--  -->
<launch>
	<node pkg="data_collect" type="float32stamped_saver" name="float32stamped_saver_steer_node" output="screen" clear_params="true">
		<param name="float32stamped_topic" type="string" value="/steer_can" />
		<param name="output_path" type="string" value="$(find data_collect)/output/steer/" />
	</node>
	
	<node pkg="data_collect" type="float32stamped_saver" name="float32stamped_saver_speed_node" output="screen" clear_params="true">
		<param name="float32stamped_topic" type="string" value="/speed_can" />
		<param name="output_path" type="string" value="$(find data_collect)/output/speed/" />
	</node>
</launch>

